src/PHPMailer.php

```diff
@@ -4476,9 +4476,11 @@ public function DKIM_HeaderC($signHeader)
         //@see https://tools.ietf.org/html/rfc5322#section-2.2
         //That means this may break if you do something daft like put vertical tabs in your headers.
         //Unfold header lines
-        $signHeader = preg_replace('/\r\n[ \t]+/', ' ', $signHeader);
+        $signHeader = str_replace(["\r\n","\r"], ["\n","\n"], $signHeader);
+        $signHeader = preg_replace('/\n[ \t]+/', ' ', $signHeader);
+        
```

Comment:
I don't think these changes are really appropriate. As the comment on this function says:

> Canonicalized headers should *always* use CRLF, regardless of mailer setting

PHPMailer tries to handle *everything* as `CRLF` until it is not required to do so (e.g. to use `mail()`). Inconsistent internal line break handling is something that was a major problem in 5.2 and was addressed in 6. So while this might fix your current issue (though I think that's strange because CRLF is *required* to calculate a DKIM signature correctly, even if the message is sent using `mail()`), I think it's likely to cause problems in other situations. A receiver will *always* recalculate the signature using CRLF, so not doing the same when sending is asking for trouble.